### DISCUSSION TITLE
Computational Constraints and Parameters of the Vehicle

### DISCUSSION THREAD

**Discussion by MrKrabb** [DLR-VSDC/IEEE-MVC-2023#3] [Q&A] [answered]
I would like to ask if the parameters like the ones in table 1 in the given paper "IEEE VTS Motor Vehicles Challenge 2023: A Multi-physical Benchmark Problem for Next Generation Energy Management Algorithms" are given somewhere or if we should identify those. In my case I need them for a model based control concept.
Second I would like to ask if there are any constraints regarding the computational effort for the Energy Management System.
EDIT: One More Question: Which signals out of the Simulink environment are allowed to be used to create the EMS? Is everything allowed or just the single Components of the total Cost value or just the Final Cost value for the whole track?

**Comment by JBRDLR** [comment, marked as answer]
Dear @MrKrabb, we did not give the parameters, since we did not give the complete equation of all model components, so it is seen more as a black box model. As a hint, you can browse the FMU in especial all *parameters where the component specific values are stored. So you could do a reverse engineering. E.g.
<img width="408" alt="image" src="https://user-images.githubusercontent.com/110536277/208142084-c5e4b30d-29d0-43c6-af2c-0a0dbc4a226d.png">


Your EMA algortihm should run all tests "batchRunMain.m" within less the 2h on a standard PC. This is not a hard constrain but otherwise we are not able to proof your controller concept.

Your EMA algortihm should match the in and outputs as they are used in the baselineEMA (see also paper / presentation).

**Comment by rpintodecastro** [comment]
Hi,

Did you try to clone the repository?
***@***.***

We received reports that the option “download ZIP” does not work with the tracks.

Best
Ricardo

**Reply by SimShady93** [reply]
Yes indeed, download zip was the culprit, to test the theory I downloaded mat files of a few tracks individually and used them in my original folder and all is working fine now. 

Thanks
Shadwal

**Reply by tobolar** [reply]
@rpintodecastro  I wonder how your answear corresponds to the original question.
